x/finality/keeper/power_dist_change.go

```diff
@@ -267,9 +284,19 @@ func (k Keeper) ProcessAllPowerDistUpdateEvents(
 		// add all BTC delegations that are not unbonded to the new finality provider
 		for j := range dc.FinalityProviders[i].BtcDels {
 			btcDel := *dc.FinalityProviders[i].BtcDels[j]
```

Comment:
Would recommend we remove any usage of `BtcDels` under `FinalityProviderDistInfo`. The original purpose of `BtcDels` is to track the share of every BTC delegator under the FP. Now this is replaced by KV stores under `x/incentive` and no need to do this anymore

Reply:
Can this be done in another PR after this one

Reply:
Could we do it in this PR or merge this PR to a feature branch? imo replacing `BtcDels` with the "virtual periods" is the main point of this feature so it's better to have it before merging this feature to `main`

Reply:
Removed `BTCDelDistInfo` structure